examples/ASMapNode/Sample/MapHandlerNode.m

```diff
@@ -235,6 +279,21 @@ - (BOOL)editableTextNode:(ASEditableTextNode *)editableTextNode shouldChangeText
   return YES;
 }
 
+- (MKAnnotationView *)annotationViewForAnnotation:(id<MKAnnotation>)annotation
+{
+  MKAnnotationView *av;
+  if ([annotation isKindOfClass:[CustomMapAnnotation class]]) {
+    av = [[MKAnnotationView alloc] init];
+    av.center = CGPointMake(21, 21);
```

Comment:
I don't think we want to set `center` here, since the map view is going to place the annotation wherever it decides. Was this meant to be `centerOffset`?